rusoto/core/src/param.rs

```diff
@@ -23,7 +26,9 @@ impl ServiceParams for Params {
     }
 }
 
+/// Trait for implementing type to paramater conversion
 pub trait ToParam {
+    /// Convert self into String
```

Comment:
Maybe it's better to say: `Renders this parameter to a String`? We're not really converting self (since we don't consume the object with: `to_param(self)`